pkg/cmd/tknpac/generate/generate.go

```diff
@@ -246,6 +246,7 @@ func (o *Opts) samplePipeline(recreateTemplate bool) error {
 	)
 	fmt.Fprintf(o.IOStreams.Out, "%s You can test your pipeline manually with: ", cs.InfoIcon())
 	fmt.Fprintf(o.IOStreams.Out, "tkn-pac resolve -f %s | kubectl create -f-\n", relpath)
```

Comment:
maybe we should remove this part since it's anyway buggy with secrets and may just be confusing with the other sentence below

Reply:
Agree